core/src/main/resources/hudson/PluginManager/check.jelly

```diff
@@ -29,5 +29,9 @@ THE SOFTWARE.
   <f:link href="${ds.useBrowser ? 'checkUpdates' : 'checkUpdatesServer'}" post="true" clazz="yui-button yui-submit-button submit-button primary">
     <button>${%Check now}</button>
   </f:link>
-
+  <j:if test="${app.pluginManager.lastErrorCheckUpdateCenters != null}">
+    <div class="error">
```

Comment:
Maybe some kind of heading explaining that the error happened when checking the update centers?